_bibliography/references.bib

```diff
@@ -1,2419 +1,4483 @@
-	language = {en-US},
-	urldate = {2020-05-22},
-	journal = {Embedded Podcast},
-	publisher = {Embedded.fm},
-	author = {White, Elecia and White, Christopher},
-	collaborator = {Huff, Kathryn D.},
-	month = may,
-	year = {2020},
-	file = {Snapshot:/Users/huff/Zotero/storage/EVY67YTW/331.html:text/html}
-}
-
-@article{miernicki_nuclear_2020,
-	title = {Nuclear waste heat use in agriculture: {History} and opportunities in the {United} {States}},
-	volume = {267},
-	issn = {0959-6526},
-	shorttitle = {Nuclear waste heat use in agriculture},
-	url = {http://www.sciencedirect.com/science/article/pii/S095965262031965X},
-	doi = {10.1016/j.jclepro.2020.121918},
-	abstract = {Nuclear power plants (NPPs) produce a large amount of waste heat (WH) that has generally been perceived and regulated as an environmental liability. Given the abundance of WH from NPPs and the ubiquity of generally low-grade heat requirements of agricultural operations, from production to post-harvest, there is remarkable potential to harness NPP WH for agricultural uses with mutual economic advantages to NPPs and agricultural sectors. Taking advantage of this WH resource may improve the financial outlook of both the partnered power plants and agricultural businesses by providing an additional revenue stream, decreased heating costs, and a reduced carbon footprint. This review summarizes and interprets the historical discourse and research on agricultural applications of NPP WH in the U.S., and synthesizes technical constraints, unknowns, and opportunities for realizing the benefits of WH derived from the nuclear energy sector for agricultural value chains. Previous applications of WH in the agricultural industry demonstrate that this is a viable option to the benefit of the parties involved under the right conditions, but relatively little has been done to further this technology in the U.S. in recent years or explore novel applications. A revival of interest in this technology may be warranted given the current outlook for NPPs in the U.S. and a general interest in reducing the environmental impact of agriculture.},
-	language = {en},
-	urldate = {2020-06-03},
-	journal = {Journal of Cleaner Production},
-	author = {Miernicki, Elizabeth A. and Heald, Alexander L. and Huff, Kathryn D. and Brooks, Caleb S. and Margenot, Andrew J.},
-	month = sep,
-	year = {2020},
-	pages = {121918},
-	file = {Elizabeth A. Miernicki et al. - 2019 - Nuclear waste heat use in agriculture history and.pdf:/Users/huff/Zotero/storage/ABSAF3SA/Elizabeth A. Miernicki et al. - 2019 - Nuclear waste heat use in agriculture history and.pdf:application/pdf;ScienceDirect Snapshot:/Users/huff/Zotero/storage/ETHXEJLI/S095965262031965X.html:text/html;ScienceDirect Full Text PDF:/Users/huff/Zotero/storage/A89NY9G3/Miernicki et al. - 2020 - Nuclear waste heat use in agriculture History and.pdf:application/pdf}
-}
+
+@phdthesis{yardas_implementation_2023,
+	type = {Thesis},

```

Comment:
```suggestion
	type = {Masters Thesis},

```
There is something like this (this may not be exactly it) you can add it to tell bibtex this is an MS thesis, or else it assumes a PhD thesis.